docs/user-guide/deep_links.md

```diff
@@ -0,0 +1,57 @@
+# Deep Links
```

Comment:
Register document in https://github.com/argoproj/argo-cd/blob/master/mkdocs.yml#L20 please